src/main/kotlin/no/nav/familie/ba/sak/behandling/BehandlingService.kt

```diff
@@ -269,6 +279,10 @@ class BehandlingService(private val behandlingRepository: BehandlingRepository,
                 .joinToString("")
     }
 
+    private fun hentSøker(behandling: Behandling): Person? {
+        return personopplysningGrunnlagRepository.findByBehandlingAndAktiv(behandling.id)!!.personer.find { person -> person.type == PersonType.SØKER }
```

Comment:
Burde vi heller bruke `?`?

Reply:
Man kan vel være rimelig sikker på at enhver aktiv behandling har et personopplysningsgrunnlag, eller?

Reply:
Japp, dere kan godt endre det slik at den ikke er nullable! 👍

Reply:
Teoretisk sett ja, men i praksis vil den vel aldri gjøre det? Kunne endret pekerne til å gå motsatt vei, men tror det er samme greia feks mellom fagsak og behandling, så da burde man gjøre det overalt og det blir en større omskriving. 

Kunne sikkert endret til `?:` her og skrevet en custom exception, men `!!` er vel ment for bruk i tilfeller hvor man er rimelig trygg på at en verdi i praksis aldri vil være null, som her?

Reply:
Litt usikker, men tror du ikke `?` vil gi bedre feilmelding? 
På tynn is her altså, men ville tro at kotlin likte bedre `?` enn `!!` fordi jeg ser for meg at den gir blanke når man putter inn `!!`. 

```suggestion
        return personopplysningGrunnlagRepository.findByBehandlingAndAktiv(behandling.id)?.personer.find { person -> person.type == PersonType.SØKER }
```

Reply:
Jeg tror `?` vil forflytte feilen til et annet sted i koden, mens `!!` vil gi en umiddelbar NPE. Et eller annet sted må man enforce at personen må eksistere, for personident-feltet mot oppdrag er påkrevd.